modules/book.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable  */
```

Comment:
- [ ] Hi, well done so far. I would advise you not to disable Linter, and try to solve the errors manually:
- A linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs(source: [Wikipedia](https://en.wikipedia.org/wiki/Lint_(software))).

There are a few reasons for using linters:

Catching syntax errors is more efficient. There is no need to debug simple mistakes like typos - the linter does it for you.
The entire codebase looks like written by one person.
Programmers can focus on solving problems, instead of cleaning up the code.

- [ ] **I would suggest that you check if your App is working well before submitting it for re-review.**